crates/llm/examples/drop_multiple_sessions.rs

```diff
@@ -0,0 +1,57 @@
+use llm::ModelArchitecture;
+use llm_base::{InferenceFeedback, InferenceParameters, ModelParameters};
+use std::{convert::Infallible, path::PathBuf};
+
+fn main() {
+    let prompt = "What is the meaning of life?";
+    let model_path = PathBuf::from(r"C:\Users\lkreu\Downloads\orca-mini-v2_7b.ggmlv3.q5_K_M.bin");
```

Comment:
Use clap, like the other examples, to avoid hardcoding a path

Reply:
I'll accidentally committed this. Will remove it before merging.